draft/spec/index.md

```diff
@@ -717,9 +717,17 @@ either a directory hierarchy of OCFL Objects or top-level OCFL Objects
 The behavior of the storage root may be extended to support features from other specifications.
 
 The base directory of an OCFL Storage Root <span class="rfc2119">MAY</span> contain a directory named `extensions` for
-the purposes of extending the functionality of an OCFL Storage Root. The storage root `extensions` directory <span
-id="E086" class="rfc2119">MUST</span> conform to the same guidelines and limitations as those defined for [object
-extensions](#object-extensions).
+the purposes of extending the functionality of an OCFL Storage Root. The guidelines and limitations for the storage
+root `extensions` directory are defined in alignment with those of the [object extensions](#object-extensions).
+
+The `extensions` directory <span id="E112" class="rfc2119">MUST NOT</span> contain any files, and no sub-directories
+other than extension sub-directories. Extension sub-directories <span id="W016" class="rfc2119">SHOULD</span> be named
+according to a <a>registered extension name</a>.
+
+> Non-normative note: Extension sub-directories should use the same name as a registered extension in order to both
```

Comment:
Can we run the two non-normative notes (this and the existing one that follows) into one? I.e. put a `>` for the blank line between paras and remove the second "Non-normative note: "

Reply:
(otherwise looks good)